commands/release.go

```diff
@@ -103,8 +102,7 @@ With '--exclude-prereleases', exclude non-stable releases from the listing.
 		does not already exist (default: main branch).
 	
 	-i, --include <PATTERN>
-		Display the files that match the prodived glob when looking at the
-		release files.	
+		Filter the files in the release to those that match the <PATTERN>.
```

Comment:
Could we clarify that this is a glob pattern? E.g.

> Download only the assets whose name matches the glob \<PATTERN>.

Reply:
yep, added that in.